app/spdb/build.sbt

```diff
@@ -22,22 +22,8 @@ ocsAppManifest := {
     configs = List(
       common(v), // note, each config extends xxx_credentials, defined in $TOP/project/OcsCredentials.scala
         with_gogo(v),
-          rnorris(v),
```

Comment:
Why are we deleting all these configurations?

Reply:
Oh crap... well, this is a throw-away PR, so it doesn't really matter.
I just removed them to speed up compilation. I don't need all of your ODBs built on my machine.